docker-compose/study/docker-compose.override.yml

```diff
@@ -107,15 +107,15 @@ services:
         condition: "service_started"
         required: false
     environment:
-      - JAVA_TOOL_OPTIONS=-Xmx576m
+      - JAVA_TOOL_OPTIONS=-Xmx1024m
```

Comment:
there is no corresponding template matching this size
The closest is size-xl: Xmx1086m/limit: 1664m
You also need to modify it in k8s/resources/study/network-modification-server-deployment.yaml so that it's accounted for in the k8s deployment